### ISSUE TITLE
semantic label is not the same inside the superpoint

### ISSUE THREAD

**Issue by Pixie412** [liuzhengzhe/One-Thing-One-Click#25]
Hi,
thanks for releasing the code. I have q question regarding the superpoint of s3dis you used in the paper. I found that the semantic label inside the superpoint is not same. In this way, how to calculate the label for the superpoint?

**Comment by liuzhengzhe** [comment]
Thanks for your question. We label one point per voxel, and expand the
labeled category to the whole super voxel.

In our experience, we simulate the above annotation process by randomly
choose one point in the super voxel, and use its semantic category to
expand to the whole super voxel.

Zhengzhe

Pixie412 ***@***.***> 于2022年9月15日周四 15:13写道：

**Comment by Pixie412** [comment]
but the labels for all the points inside a superpoint may not belong to the same category. In my project, I empirically found that this affects the performance compared to directly predicting label for each point. 
Did you encounter similar problem?

**Comment by liuzhengzhe** [comment]
Yes. Our setting is to only annotate one point per category, so the GT is
not perfect and it of course brings some error.

As for performance, after the super-voxel pooling, the performance will be
improved by around 1% to 2% compared with point-wise prediction.

Zhengzhe

Pixie412 ***@***.***> 于2022年9月15日周四 15:59写道：
